qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -1004,6 +1004,10 @@ def run(
                 instance=qrt_options.instance, backend_name=qrt_options.backend
             )
             hgp_name = hgp.name
+        backend = self.backend(name=qrt_options.backend, instance=hgp_name)
+        status = backend.status()
+        if status.operational is True and not status.status_msg == "active":
```

Comment:
nit: is this equivalent to 
```suggestion
        if status.operational is True and status.status_msg != "active":
```

I think no test for this would be ok